Dockerfile

```diff
@@ -196,16 +206,19 @@ RUN curl --fail -sSL -o /usr/local/bin/goofys https://github.com/kahing/goofys/r
 #
 # Install Google Cloud SDK
 #
-ENV GCLOUD_SDK_VERSION=179.0.0
-RUN curl --fail -sSL -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${GCLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
+ONBUILD ARG GCLOUD_SDK_VERSION
+#179.0.0
```

Comment:
I'm torn. I think it might be best to let the caller *disable* a feature by setting a nil arg, rather than leaving it disabled by default.

Reply:
Works that way also, or you can cherry pick the behavior. Some needs enabling e.g Ansible (else by default it will be skipped) on the other hand (some might need disabling).
This would make script bit complicated though as you need to keen relationship between 2.
Let's say a user can use dep A, but if he wants dep B (A can't be excluded ). This kind of scenario becomes tricky[though remember YAGNI]. Another thing which I want to do but couldn't do is, if we provide the only arg as KOPS_VERSION[use some pinned persion], if we use KOPS_VERSION=1.9, use this version. Something like requirements.txt kind of behavior.